models/org.go

```diff
@@ -453,6 +455,8 @@ func (org *User) getUserTeams(e Engine, userID int64, cols ...string) ([]*Team,
 	return teams, e.Where("team_user.org_id = ?", org.ID).
 		And("team_user.uid = ?", userID).
 		Join("INNER", "team_user", "team_user.team_id = team.id").
+		Join("INNER", "user", `"user".id=team_user.uid`).
```

Comment:
Hmm.. Ideally, we should not quote column names. Quoting is different between different databases. PostgreSQL uses ", MySQL uses `, etc. I think using [this Xorm function][quote] should work.

Same for other SQLs.

[quote]: https://godoc.org/github.com/go-xorm/xorm#Engine.QuoteStr